src/bin/CMakeLists.txt

```diff
@@ -118,12 +122,21 @@ if(NOT WIN32)
     # otherwise.
     #
     # this is copying to build dir
+    set(__all_visit_bin_build_dir_items)
     foreach(VISIT_BIN_ITEM ${VISIT_BIN_FILES} internallauncher makemovie.py makemoviemain.py visitcinema.py visitcinemamain.py visitdiff.py)
-        execute_process(COMMAND ${CMAKE_COMMAND} -E copy
-                                ${CMAKE_CURRENT_SOURCE_DIR}/${VISIT_BIN_ITEM}
-                                ${CMAKE_CURRENT_BINARY_DIR}/${VISIT_BIN_ITEM})
+        # use custom command so cmake captures the dependency, triggering
+        # a copy whenever the source files change
```

Comment:
just curious if using the previous command but changing it to 'copy_if_different' would have sufficed?

Otherwise, would suggest copy_if_different in the custom command.  Not that it will affect performance all that much given the small number of files being copied in this instance.  Its just I see every bit of lag when building on Windows.

Reply:
Actually I am wrong, the custom command is needed, as the previous is only executed at cmake configure not during build and subsequent rebuilds.